mutt/prex.c

```diff
@@ -121,29 +121,52 @@ static struct PrexStorage *prex(enum Prex which)
       PREX_RFC5322_DATE,
       PREX_RFC5322_DATE_MATCH_MAX,
       /* Spec: https://tools.ietf.org/html/rfc5322#section-3.3 */
-#define C "" //"\\([^()\\]*\\)?" // Comment are obsolete but still allowed
       "^"
         "("
-        C " *"
-          "(Mon|Tue|Wed|Thu|Fri|Sat|Sun)"  // Day of week
-        C ", "
+          "(Mon|Tue|Wed|Thu|Fri|Sat|Sun)" // Day of week
```

Comment:
There are now two copies of `(Mon|Tue|...)` and three `(Jan|Feb|...)`
Should we adopt your `PREX_DOW`, `PREX_MONTH`, etc

Reply:
yes, that's coming in the other PR. I can bring them here or we can wait for that one to be finished.

Reply:
I'll wait